src/Sentry/Protocol/Metrics/CounterMetric.cs

```diff
@@ -26,7 +26,7 @@ public CounterMetric()
     protected override void WriteValues(Utf8JsonWriter writer, IDiagnosticLogger? logger) =>
         writer.WriteNumber("value", Value);
 
-    protected override IEnumerable<IConvertible> SerializedStatsdValues()
+    protected override IEnumerable<object> SerializedStatsdValues()
```

Comment:
IConvertible isn't  CLS Compliant so it can't be in any public signatures for libraries targeting netstandard 2.x.

Reply:
We could drop CLS compliance on 4.0. But I'm not sure IConvertible is really used anymore anyway is it?